lib/Subscriber/Core/MappingSubscriber.php

```diff
@@ -222,7 +264,8 @@ private function hydrateReferenceField(
         DocumentAccessor $accessor,
         $fieldName,
         $locale,
-        array $fieldMapping
+        array $fieldMapping,
+        array $options
```

Comment:
I think we should always propagate the options to the proxy. This was likely an existing bug /cc @wachterjohannes